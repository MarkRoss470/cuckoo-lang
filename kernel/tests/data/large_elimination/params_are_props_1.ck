data SomeProp1 : Prop where
  | c1 : SomeProp1
  | c2 : SomeProp1

data SomeProp2 : Prop where
  | c : SomeProp1 -> SomeProp2

-- This `Prop` judgement is wrong, but never reached because the level error is encountered first
def some_prop_rec : Prop := SomeProp2.rec