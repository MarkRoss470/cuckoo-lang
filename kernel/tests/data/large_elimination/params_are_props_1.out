ERROR at tests/data/large_elimination/params_are_props_1.ck 9:29-9:42: SomeProp2.rec takes 1 level argument(s), but 0 were provided.